txstratum/api.py

```diff
@@ -157,7 +157,8 @@ async def submit_job(self, request: web.Request) -> web.Response:
             self.manager.add_job(job)
         except JobAlreadyExists:
             self.log.debug("job-already-exists", data=data)
-            return web.json_response({"error": "job-already-exists"}, status=400)
```

Comment:
It seems the current set of tests does not cover this part. I feel we should add tests to it.